### PR TITLE
Transitive classes

### PR DESCRIPTION
**Pull request by benjub**
Commit messages say it all.

### CODE REVIEW

File: set.mm [hunk 1]
```diff
@@ -206874,14 +206907,14 @@ that F(A') is isomorphic to B'.".  Therefore, the category of sets and
       UIUQDVACSUJUKUSERZVRWAVHWCVPVTVQWCVOVIVBVSUSEDCTUSEVACSUKULUMVAFRZWAVMVHW
       DVTVKVQVLWDVSVJVIVAFECTUMVAFDCTUJUMUNUO $.
 
-    $( Less-or-equal is reflexive in a proset.  (Contributed by Stefan O'Rear,
-       1-Feb-2015.) $)
+    $( Less-than-or-equal-to is reflexive in a proset.  (Contributed by Stefan
```
Comment (avekens) [inline set.mm:206910]: Why not "Less than or equal to" as above?

File: set.mm [hunk 2]
```diff
@@ -707210,8 +707244,8 @@ Ordering on reals (cont.) - extension
 -.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-.-
 $)
 
-  $( Addition and subtraction on one side of 'less or equal'.  (Contributed by
-     Alexander van der Vekens, 18-Mar-2018.) $)
+  $( Addition and subtraction on one side of "less-than-or-equal-to".
```
Comment (avekens) [inline set.mm:707247]: Either "less than or equal to" or less-than-or-equal-to.

**Review by avekens — CHANGES_REQUESTED** [review]
Only a mior remark: Either "less than or equal to" or less-than-or-equal-to should be used.

File: set.mm (hunk 2 above)
Comment (benjub) [reply set.mm:707247]: Done.

File: set.mm (hunk 1 above)
Comment (benjub) [reply set.mm:206910]: I don't know ! I found both ways in set.mm and wasn't sure which was best and if I was a bit too "dictatorial" in enforcing one or the other. What's important is that I fixed the few "less or equal to..." to "less than or equal to". If we want to enforce a consistent spelling, we can do it in a future PR.

**Review by avekens — APPROVED** [review]
Less-than-or-equal-to should be changed to "Less than or equal to" in a leter PR.

**Comment by benjub** [comment]
> Less-than-or-equal-to should be changed to "Less than or equal to" in a leter PR.

Ok, I'm doing it now.

**Comment by benjub** [comment]
Ok, I did it in the latest (hopefully last) commit. Though I'm not convinced by it. Now, there are still many occurrences of `less-than` that we should also change to `"less than"`, and there is also the `at-most-one quantifier`... I think we can leave it that way. At least we reduced these expressions to only two versions, so it's a bit easier to search.

**Review by avekens — APPROVED** [review]

**Pull request merged**